src/Plane.js

```diff
@@ -0,0 +1,8 @@
+class Plane {
```

Comment:
Added name constructor to Plane so that names could be set on the instances (I had initally been trying to use names in the Runner without setting a name constructor...)